web/src/components/PasswordWithMeter.test.js

```diff
@@ -2,28 +2,36 @@ import { shallow } from 'enzyme';
 import React from 'react';
 import sinon from 'sinon';
 
+// Import the lazy-loading zxcvbn so we can use jest's mocking functionality
+// eslint-disable-next-line no-unused-vars
+import zxcvbn from '../lazy/zxcvbn';
+
 import Password from './PasswordWithMeter';
 
+jest.mock('../lazy/zxcvbn');
+
 describe('PasswordWithmeter component', () => {
   it('renders with no props', () => {
     expect(shallow(<Password />)).toMatchSnapshot();
   });
 
   it('renders with various passwords', () => {
     // weak, strength = 0
-    expect(shallow(<Password value="a" />)).toMatchSnapshot();
+    expect(shallow(<Password value="a" showMeter />)).toMatchSnapshot();
```

Comment:
It turns out these tests were also just wrong, because `showMeter` defaults to `false`, so all the snapshots were identical - no meter.  😬

Reply:
Ahh, that's my bad!